bpf/cpu/cpu.bpf.c

```diff
@@ -818,12 +843,18 @@ int walk_user_stacktrace_impl(struct bpf_perf_event_data *ctx) {
       return 1;
     }
 
+    // lr offset is only fetched from userspace and used as a field in unwind table for Arm64
+    #if __TARGET_ARCH_arm64
+      s16 found_lr_offset = unwind_table->rows[table_idx].lr_offset;
+    #else
+      s16 found_lr_offset = 0;
+    #endif
     u64 found_pc = unwind_table->rows[table_idx].pc;
     u8 found_cfa_type = unwind_table->rows[table_idx].cfa_type;
     u8 found_rbp_type = unwind_table->rows[table_idx].rbp_type;
     s16 found_cfa_offset = unwind_table->rows[table_idx].cfa_offset;
     s16 found_rbp_offset = unwind_table->rows[table_idx].rbp_offset;
-    LOG("\tcfa type: %d, offset: %d (row pc: %llx)", found_cfa_type, found_cfa_offset, found_pc);
+    LOG("\tcfa type: %d, offset: %d (row pc: %llx), lr offset:%d", found_cfa_type, found_cfa_offset, found_pc, found_lr_offset);
```

Comment:
Just to accelerate debugging, this is causing this error in the 2 older kernels we support:

```
2023-09-18T14:49:20.1466024Z [            ] stderr: 524: (67) r1 <<= 48
2023-09-18T14:49:20.1468994Z [            ] stderr: 525: (c7) r1 s>>= 48
2023-09-18T14:49:20.1472421Z [            ] stderr: 526: (7b) *(u64 *)(r10 -48) = r1
2023-09-18T14:49:20.1475730Z [            ] stderr: 527: (bf) r3 = r10
2023-09-18T14:49:20.1485343Z [            ] stderr: ; 
2023-09-18T14:49:20.1530281Z [            ] stderr: 528: (07) r3 += -56
2023-09-18T14:49:20.1536686Z [            ] stderr: ; LOG("\tcfa type: %d, offset: %d (row pc: %llx), lr offset:%d", found_cfa_type, found_cfa_offset, found_pc, found_lr_offset);
2023-09-18T14:49:20.1540896Z [            ] stderr: 529: (18) r1 = 0xffff8ca5102ea3ba
2023-09-18T14:49:20.1543528Z [            ] stderr: 531: (b7) r2 = 60
2023-09-18T14:49:20.1547193Z [            ] stderr: 532: (b7) r4 = 32
2023-09-18T14:49:20.1550524Z [            ] stderr: 533: (bf) r6 = r0
2023-09-18T14:49:20.1553673Z [            ] stderr: 534: (85) call unknown#177
2023-09-18T14:49:20.1556923Z [            ] stderr: invalid func unknown#177
```

This is due to the `bpf_printk` choosing to use `bpf_trace_vprintk` due to the number of arguments being above some threshold. `bpf_trace_vprintk`, which is not supported in older kernels.

This is implemented in https://github.com/libbpf/libbpf/blob/56069cda7897afdd0ae2478825845c7a7308c878/src/bpf_helpers.h#L288-L295